test/package.json

```diff
@@ -40,6 +41,7 @@
     "@dfinity/candid": "^0.11.1",
     "@dfinity/identity": "^0.11.1",
     "@dfinity/principal": "^0.11.1",
```

Comment:
Is it because of cap-js?

Reply:
Yeah, apparently dfinity changed the principal _arr again between 0.11 and 0.10 😭